server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -61,6 +58,9 @@ public class KillUnusedSegments implements CoordinatorDuty
 {
   public static final String KILL_TASK_TYPE = "kill";
   public static final String TASK_ID_PREFIX = "coordinator-issued";
+  public static final Predicate<TaskStatusPlus> TASK_PREDICATE =
```

Comment:
Similar here, I think we should give the name a more clear name, `IS_KILL_TASK`?

Reply:
fixed